### PR TITLE
HieuAddGameOverUI

### PR DESCRIPTION
**Pull request by NgoHieu-coder**

### CODE REVIEW

**Comment by QuangWorkIT** [comment]
1. Solve conflict
2. Đổi lại background
3. Dùng hàm changeScene của Viên đổi tên biến ko cần mỗi scene 1 hàm

File: Assets/BG_Galaxy.jpg
Comment (AnataAria) [inline Assets/BG_Galaxy.jpg:1]: Use the original image provide file by @QuangWorkIT , if you have other reason why u use this, please comment here and btw make a folder name Sprites and put all assets like image to it , don't put it to root folder, that will made a mess

File: Assets/BG_Galaxy.jpg.meta
Comment (AnataAria) [inline Assets/BG_Galaxy.jpg.meta:1]: Same as above

File: Assets/GOAsset.meta
Comment (AnataAria) [inline Assets/GOAsset.meta:1]: Save as above

File: Assets/Script/Scene_Script/changeScene.cs
Comment (AnataAria) [inline Assets/Script/Scene_Script/changeScene.cs:1]: File name and class name wrong syntax, should use camel case, not this, and use code of @TDVien05 , contact him for more detail

Comment (AnataAria) [inline Assets/Script/Scene_Script/changeScene.cs:1]: And Folder Scene_Scripts should not name like this, that will more unreasonable bugs in future and hard to read, remove underline

File: Assets/Script/Scene_Script/changeScene.cs [hunk 1]
```diff
@@ -10,4 +10,39 @@ public void change()
     {
         SceneManager.LoadScene(sceneName);
     }
```
Comment (AnataAria) [inline Assets/Script/Scene_Script/changeScene.cs]: I dont think that will Ok, wrong name convention

**Review by AnataAria — CHANGES_REQUESTED** [review]

File: Assets/BG_Galaxy.jpg
Comment (QuangWorkIT) [inline Assets/BG_Galaxy.jpg:1]: Change the background according to the UI folder

**Pull request closed without merging**